lib/hider/accounts.ex

```diff
@@ -21,6 +23,20 @@ defmodule Hider.Accounts do
     Repo.all(User)
   end
 
+  @doc """
+  Fuzzy Full Text Search for user
+  """
+  def fuzzy_search(text) do
```

Comment:
Make the search trigrams and hash them all to match agains the database.
This can be optimized to not encrypt all the trigrams only some, and try, then if nothing happens try all of then (???)